fontbe/src/gvar.rs

```diff
@@ -52,6 +52,14 @@ impl Work<Context, AnyWorkId, Error> for GvarWork {
         // We built the gvar fragments alongside glyphs, now we need to glue them together into a gvar table
         let static_metadata = context.ir.static_metadata.get();
         let axis_order: Vec<_> = static_metadata.axes.iter().map(|a| a.tag).collect();
+        let axis_count: u16 = axis_order
+            .len()
+            .try_into()
+            // in the unlikely event that we have more than 65535 axes...
+            .map_err(|_| Error::OutOfBounds {
+                what: "axis count".into(),
+                value: axis_order.len().to_string(),
+            })?;
```

Comment:
I would personally unwrap here, since I can't imagine this being possible for any conceivable font, but 🤷

Reply:
just wait a few years :)

Reply:
I strongly prefer Result unless it _can't_ happen. Rationale: if we unwrap all the things that shouldn't happen but plausibly can then it's all but assured we'll hit _some_ of them in production and rue the day we didn't use Result.